gop.mod

```diff
@@ -0,0 +1,3 @@
+gop 1.3
```

Comment:
don't need `gop.mod`, use `require github.com/goplus/spx v1.1.0 //gop:class` in `go.mod`.